components-compose/src/main/java/uk/gov/hmrc/components/compose/organism/primary/PrimaryCardView.kt

```diff
@@ -15,4 +15,96 @@
  */
 package uk.gov.hmrc.components.compose.organism.primary
 
-class PrimaryCardView
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import uk.gov.hmrc.components.compose.atom.heading.Heading5
+import uk.gov.hmrc.components.compose.organism.HmrcCardView
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
+
+@Composable
+fun PrimaryCardView(
+    modifier: Modifier = Modifier,
+    title: String,
+    titleContentDescription: String = "",
+    childPadding: Boolean = true,
+    content: @Composable (ColumnScope.() -> Unit)? = null
+) {
+    val childPaddingState by remember { derivedStateOf { childPadding } }
+
+    HmrcCardView(modifier = modifier) {
+        Heading5(
+            text = title,
+            modifier = Modifier
+                .padding(
+                    start = HmrcTheme.dimensions.hmrcSpacing16,
+                    end = HmrcTheme.dimensions.hmrcSpacing16,
+                    top = HmrcTheme.dimensions.hmrcSpacing16
+                )
+                .semantics(mergeDescendants = true) {
+                    contentDescription = titleContentDescription
+                }
+                .fillMaxWidth()
+        )
+        if (content != null) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .then(
+                        if (childPaddingState) {
+                            Modifier
+                                .padding(HmrcTheme.dimensions.hmrcSpacing16)
+                        } else {
+                            Modifier
+                        }
+                    )
+            ) {
+                content()
+            }
+        }
+    }
+}
+
+@Preview
+@Composable
+fun PrimaryCardViewPreviewEmpty() {
+    HmrcTheme() {
+        PrimaryCardView(
+            modifier = Modifier,
+            title = "",
+            content = {
+                Text(
+                    text = "",
+                    style = HmrcTheme.typography.body
+                )
+            }
+        )
+    }
+}
```

Comment:
This doesn't look like representative usage I don't think.. not sure there's any scenario we would be adding a Text child with no string provided? I think the baseline example would be a primary card with a title but no children...
```suggestion
fun PrimaryCardViewPreviewEmpty() {
    HmrcTheme() {
        PrimaryCardView(title = "Title")
    }
}
```